### DISCUSSION TITLE
Not able to run wagtail bakerydemo on windows

### DISCUSSION THREAD

**Discussion by dhruvpal05** [wagtail/bakerydemo#471] [Q&A]
I have completed all the steps mentioned in "setting up the wagtail codebase" compiled the assets successfully also ran `pip install -e path/to/wagtail"[testing, docs]" -U` this command to install `wagtail-6.0a0` , then I tried running the bakerydemo and it shows no such file or directory to `docker-endpoint.sh` , I have attached a screenshot if anyone can guide me through it would be great 

![Screenshot (405)](https://github.com/wagtail/wagtail/assets/91111849/a52b2737-770d-40fa-8ce1-723008d7d9d1)

**Comment by zerolab** [comment]
This is a bakerydemo issue, not a Wagtail one. See https://github.com/wagtail/bakerydemo/issues/468. Transferring the discussion to the bakerydemo repository

**Comment by Amondale** [comment]
This is a common problem with Windows Line Endings (CRLF) and converting to Unix (LF) line endings with your favorite tool (dos2unix) or editor (Notepad++ is my favorite) has done the trick for me. If you use windows versions of git also it seems to mess with the line endings.
